autoload/go/tool.vim

```diff
@@ -175,7 +175,7 @@ function! go#tool#ExecuteInDir(cmd) abort
   let dir = getcwd()
   try
     execute cd . fnameescape(expand("%:p:h"))
-    let [l:out, l:err] = go#util#Exec(a:cmd)
+    let [l:out, l:err] = go#util#ExecSystem(a:cmd)
```

Comment:
`ExecuteInDir` is used for other tools, too, and shouldn't be changed to use `ExecSystem` instead of `Exec`.